### PR TITLE
Tiny improvement security

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
A tiny improvement to SECURITY.md

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 4d663f36737751d2bff52ac3fd4bbf0992f5cc46 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/57171/4d663f36737751d2bff52ac3fd4bbf0992f5cc46/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/57171/4d663f36737751d2bff52ac3fd4bbf0992f5cc46/ci_running.html">✅ success</a></td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6974380074">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: SECURITY.md [hunk 1]
```diff
@@ -21,8 +21,7 @@ The following versions of ClickHouse server are currently being supported with s
 | 23.5 | ❌ |
 | 23.4 | ❌ |
 | 23.3 | ✔️ |
-| 23.2 | ❌ |
```
Comment (alexey-milovidov) [inline SECURITY.md:24]: This makes it ambiguous.

Comment (Felixoid) [reply SECURITY.md:24]: so, when `23.8` will remain the last supported version in 23, it's better to have all the rest of the year?

Comment (alexey-milovidov) [reply SECURITY.md:24]: Yes. No ambiguity.

Comment (Felixoid) [reply SECURITY.md:24]: Ok, I'll then only improve readability

**Comment by alexey-milovidov** [comment]
@Felixoid If you say "23.3 is supported" and "23.* is unsupported" it is controversial, it can be put out of context, and generally not good. Being as precise as possible is better.

**Comment by Felixoid** [comment]
Sure, I've brought them back. I meant the script readability.

As well, I'll backport it to release branches to have `.dockerignore` removed there. It's necessary for #31473

**Comment by Felixoid** [comment]
It's checked by style check, and I've tested all docker images from this branches for tags from 22.3.20.29 to 23.10.4.25

**Pull request merged**